### PR TITLE
Add exercise: protein-translation 🧬 

### PR DESCRIPTION
**Pull request by AnneKlapwijk**
## Link to forum posts

- https://forum.exercism.org/t/kotlin-protein-translation-exercise-48in24/9605
- https://forum.exercism.org/t/48in24-exercise-02-13-protein-translation/9556

## What has been done
- Use `configlet` to generate common files
- Update some config files
- Implement test cases (got the idea for a parameterized test from the JS track: https://github.com/exercism/javascript/blob/main/exercises/practice/protein-translation/protein-translation.spec.js#L22)
- Write stub implementation: provide a starting point for students
- Write an example solution that passes all the tests
  - `./gradlew practice:protein-translation:test`

## Notes
First PR here. Let me know if I missed anything 😊

### CODE REVIEW

**Comment by AnneKlapwijk** [comment]
Ping @exercism-kotlin 🙂 🛎️

File: exercises/settings.gradle.kts [hunk 1]
```diff
@@ -53,6 +53,7 @@ include(
     "practice:pangram",
     "practice:pascals-triangle",
     "practice:perfect-numbers",
+    "practice:protein-translation",
```
Comment (SleeplessByte) [inline exercises/settings.gradle.kts:56]: This list looks lexicographical, so could you move this below `"practice:prime-factors"`, so the order is preserved?

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
Maar 1 kleine opmerking

File: exercises/settings.gradle.kts (hunk 1 above)
Comment (AnneKlapwijk) [reply exercises/settings.gradle.kts]: Well spotted! Thanks for reviewing! 😊

**Review by SleeplessByte — APPROVED** [review]

**Pull request merged**